src/scripts/eastern_kingdoms/dun_morogh/gnomeregan/gnomeregan.cpp

```diff
@@ -35,33 +35,33 @@ EndContentData */
 
 enum
 {
-    SAY_START                   = -1090000,
-    SAY_INTRO_1                 = -1090001,
-    SAY_INTRO_2                 = -1090002,
-    SAY_INTRO_3                 = -1090003,
-    SAY_INTRO_4                 = -1090004,
-    SAY_LOOK_1                  = -1090005,
-    SAY_HEAR_1                  = -1090006,
-    SAY_AGGRO_1                 = -1090007,
-    SAY_CHARGE_1                = -1090008,
-    SAY_CHARGE_2                = -1090009,
-    SAY_BLOW_1_10               = -1090010,
-    SAY_BLOW_1_5                = -1090011,
-    SAY_BLOW_1                  = -1090012,
-    SAY_FINISH_1                = -1090013,
-    SAY_LOOK_2                  = -1090014,
-    SAY_HEAR_2                  = -1090015,
-    SAY_CHARGE_3                = -1090016,
-    SAY_CHARGE_4                = -1090017,
-    SAY_BLOW_2_10               = -1090018,
-    SAY_BLOW_2_5                = -1090019,
-    SAY_BLOW_SOON               = -1090020,
-    SAY_BLOW_2                  = -1090021,
-    SAY_FINISH_2                = -1090022,
-
-    SAY_AGGRO_2                 = -1090028,
-
-    SAY_GRUBBIS_SPAWN           = -1090023,
```

Comment:
Script text -1090023 has `emote` = 1 (Talk), while broadcast text id 4328 which u replace it with currently has no emote, but you do not update the broadcast text in your migration.